bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyRelayHandler.java

```diff
@@ -145,6 +145,7 @@ public boolean handleDeviceCommand(ChannelUID channelUID, Command command) throw
     }
 
     /**
+     * PaperUI Control has a combined Slider for Brightness combined with On/Off
```

Comment:
I don't think we should mention PaperUI anymore.

Reply:
changed